public/lib/latencyBasedRouting.js

```diff
@@ -143,6 +150,18 @@
         self.latencyHttpTestRequest.push(latencyHttpTestSuite);
     };
 
+    latencyBasedRouting.prototype.monitor = function () {
+
+        if (Date.now() - this.beginTime > this.latencyBasedRoutingTimeout) {
+            clearInterval(this.interval);
+            if (this.numServersResponded >= 1) {
```

Comment:
You may need to sort here, just to make sure, as we may have received 2 servers.